01/ronaldokun/wordvalue.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from data import DICTIONARY, LETTER_SCORES
+
+char = re.compile(r"[A-Z]{1}")  # I want to match individual UPPER CASE LETTERS
```

Comment:
Does this include for example german "ß"? :p Be careful with hardcoded allowed signed. It is much better to use the provided DICTIONARY with `get` method which will default to a default value for an invalid key. Dont try to predict all valid keys